CHANGELOG

```diff
@@ -13,6 +13,9 @@ Changes are grouped as follows
 - `Security` in case of vulnerabilities.
 
 
+# [2.3.10] - 2023-11-22
+- TODO: write down the changes
```

Comment:
Shouldn't we state here that `cognite-sdk` version was bumped?

Reply:
Along with other things, ideally yes